packages/core/src/App/Containers/app-notification-messages.jsx

```diff
@@ -36,11 +36,21 @@ const NotificationsContent = ({ style, notifications, removeNotificationMessage
     </div>
 );
 
-const AppNotificationMessages = ({ marked_notifications, notification_messages, removeNotificationMessage }) => {
+const AppNotificationMessages = ({
+    is_notification_loaded,
+    is_mt5,
+    marked_notifications,
+    notification_messages,
+    removeNotificationMessage,
+    stopNotificationLoading,
+}) => {
     const [style, setStyle] = React.useState({});
     const [notifications_ref, setNotificationsRef] = React.useState(null);
 
     React.useEffect(() => {
+        if (is_mt5) {
```

Comment:
(question) Any specific reason that we would want this only when `is_mt5`? Why not use it globally?

Reply:
This behaviour observed on mt_5 modal only, so I need to be specific for this case and not affect other cases.